nullaway/src/main/java/com/uber/nullaway/ErrorBuilder.java

```diff
@@ -124,6 +133,9 @@ public Description createErrorDescription(
       builder = addSuggestedSuppression(errorMessage, suggestTree, builder);
     }
     // #letbuildersbuild
```

Comment:
Comment refers to the final line of the method (though it is a bit of a joke, we could either remove the comment or move it to right before the return, but definitely not keep it up here...)